typeDefinition/ElementChain.d.ts

```diff
@@ -13,8 +13,10 @@ import {
 	BaseChain,
 	UntilModifier,
 } from './modifiers';
+import {DomElementProps} from "./constants/DomElementProps";
 
 export interface ElementChain extends
+    DomElementProps,
```

Comment:
Is it the correct place for it? It's what Element Query Promise returns, not a property of element chain.